components/ClaimSuccessPage/ClaimSuccessPage.tsx

```diff
@@ -0,0 +1,202 @@
+import { useMeasure } from "react-use"
+import { useRef } from "react"
+import { useMediaQuery, useReadLocalStorage } from "usehooks-ts"
+import Image from "next/image"
+import Layout from "../Layout"
+import SectionTitle from "../LandingPage/SectionTitle"
+import SectionContent from "../LandingPage/SectionContent"
+import { Button } from "../../shared/Button"
+import Media from "../../shared/Media"
+import Footer from "../Footer"
+import { useTheme } from "../../providers/ThemeProvider"
+import useGradualFadeEffect from "../../hooks/useGradualFade"
+
+const ClaimSuccessPage = () => {
+  const [containerRef, { width }] = useMeasure()
+
+  const isResponsive = useMediaQuery("(max-width: 1429px)")
+  const isScrollUp = useReadLocalStorage<boolean>("isScrollUp")
+  const isMobile = useMediaQuery("(max-width: 768px)")
+
+  const { themeMode } = useTheme()
+
+  const titleRef = useRef()
+  const contentRef = useRef()
+  const buttonRef = useRef()
+
+  useGradualFadeEffect({
+    elements: [
+      {
+        domObject: contentRef.current,
+        type: "child",
+      },
+      {
+        domObject: titleRef.current,
+        type: "child",
+      },
+      {
+        domObject: buttonRef.current,
+        type: "child",
+      },
+    ],
+    isScrollUp,
+  })
+
+  return (
+    <Layout type="base">
+      <div
+        className="relative overflow-y-auto min-h-[100vh] overflow-x-hidden z-[1] w-[100vw]"
+        ref={containerRef}
+      >
+        {width && (
+          <div
+            className="relative z-[6] flex flex-col items-center pt-[80px]"
+            style={{
+              width: `${width}px`,
+              height: isResponsive ? `auto` : `${(1048 / 1579) * width}px`,
+              minHeight: isResponsive ? "100vh" : "",
+              backgroundImage:
+                // eslint-disable-next-line no-nested-ternary
+                themeMode === "light"
+                  ? "url('/assets/Claim/white_background.svg')"
+                  : isMobile
+                  ? "url('/assets/Claim/mobile_dark_background.svg')"
+                  : "url('/assets/Claim/background.svg')",
+              backgroundSize: isResponsive
+                ? `cover`
+                : `${width * 1.04}px ${(1048 / 1579) * width * 1.04}px`,
+              backgroundPosition: isResponsive
+                ? `center center`
+                : `bottom 0px right -${themeMode === "light" ? 0 : width * 0.04}px`,
+            }}
```

Comment:
why are we using `style` prop?

We prefer to use Tailwind CSS `className` prop.